src/Components/Charts/Bar.tsx

```diff
@@ -0,0 +1,387 @@
+import React, { useContext, useEffect, useState } from "react";
+import { ResponsiveBar } from '@nivo/bar'
+import axios from "axios";
+import { AuthContext } from "../../Context/AuthContext";
+
+import { ResponsiveStream } from '@nivo/stream'
+export default function Bar() {
+  const { baseUrl, requstHeaders }: any = useContext(AuthContext);
+  const [contTasks, setContTasks] = useState([]);
+  const [contUsers, setcontUsers] = useState([]);
+  const getContTasks = () => {
+    axios
+      .get(`${baseUrl}/Task/count`, {
+        headers: requstHeaders,
+      })
+      .then((response) => {
+        console.log(response);
+        setContTasks(response?.data);
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  };
+
+  const getContUsers = () => {
+    axios
+      .get(`${baseUrl}/Users/count`, {
+        headers: requstHeaders,
+      })
+      .then((response) => {
+        console.log(response);
+        setcontUsers(response?.data);
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  };
+  useEffect(() => {
+    getContTasks();
+    getContUsers();
+  }, []);
+  const data = [
```

Comment:
The data used here contains date, but the data coming from the backend doesn't have any date, so we will not be able to make a similar graph from the data given from the backend.